tests/test_sft_trainer.py

```diff
@@ -190,31 +174,30 @@ def test_resume_training_from_checkpoint_with_flag_false():
         assert init_trainer_state is not None
 
         # Get Training log entry for epoch 1
-        init_training_logs = _get_training_logs(tempdir, epoch=1)
+        init_training_logs = _get_training_logs_by_epoch(tempdir, epoch=1)
         assert len(init_training_logs) == 1
 
-        # Resume training with higher epoch and same output dir
+        # Training again with higher epoch and same output dir
         train_args.num_train_epochs += 5
         sft_trainer.train(MODEL_ARGS, DATA_ARGS, train_args, None)
         _validate_training(tempdir)
 
         # Get Training log entry for epoch 1
-        final_training_logs = _get_training_logs(tempdir, epoch=1)
+        final_training_logs = _get_training_logs_by_epoch(tempdir, epoch=1)
         assert len(final_training_logs) == 2
 
 
 def test_resume_training_from_checkpoint_with_flag_checkpoint_path():
     """
-    Test feature of resume training from checkpoint using resume_from_checkpoint flag
-    to ensure that when the value of flag is a checkpoint-x path, the tuning will resume
-    from that checkpoint. This test checks if total_flos of checkpoint-x has not changed
-    after resuming tuning, hence confirming that the tuning has started from checkpoint-x.
+    Test when setting resume_from_checkpoint=path/to/checkpoint-x
+    that the tuning will resume from the checkpoint-x.
```

Comment:
nit: Could be a little clearer and should update that this is for LoRA
```suggestion
    Test resume checkpoint from a specified checkpoint path for LoRA tuning.
```

Reply:
Thank you!